src/main/java/org/recordrobotics/charger/control/DoubleControl.java

```diff
@@ -24,6 +27,25 @@ public double getDriveLat() {
 		return -_gamepad2.getLeftX();
 	}
 
+	@Override
+	public ClawState getClawTurn() {
+		// Out mimics button-like behavior
+		boolean cube = _gamepad2.getLeftTriggerAxis() > TRIGGER_THRESHOLD;
+		boolean cone = _gamepad2.getRightTriggerAxis() > TRIGGER_THRESHOLD;
+
+		if (_gamepad2.getLeftTriggerAxis() > TRIGGER_THRESHOLD)
+			_lTrig = !_lTrig;
```

Comment:
This flips the value of lTrig/rTrig every tick trigger is pushed in.
Are you sure this is correct?